### PR TITLE
Update archive/unarchived related parts in UG for v1.3

### PR DESCRIPTION
**Pull request by yanbingtao**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -299,7 +329,8 @@ Action | Format, Examples
 **Search** | `search KEYWORD [MORE_KEYWORDS]`<br> e.g., `search James Jake`
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
 **Clear** | `clear`
-**Archive** |  1. **Archive \(1 entry\):**  `archive INDEX`<br> e.g., `archive 1` <br>2. **Archive \(all\):**  `archvie all` <br>3. **List all archived data:**  `archvie list`</br>
+**Archive** |  1. **Archive \(1 employee\):**  `c-archive INDEX`<br> e.g., `c-archive 1` <br>2. **Archive \(all emp;oees\):**  `c-archvie all` <br>3. **List all archived data:**  `c-archvie list`</br>
```
Comment (WM71811) [inline docs/UserGuide.md]: **Archive \(all emp;oees\):** here should be employees?

Comment (yanbingtao) [reply docs/UserGuide.md]: Thank you! Have corrected this part.

**Review by Persdre — APPROVED** [review]
LGTM

**Review by WM71811 — APPROVED** [review]
LGTM

**Pull request merged**